README.md

```diff
@@ -1,7 +1,39 @@
-# Spectral Lint Action
+# Spectral Linter Action
 
-![](https://raw.githubusercontent.com/stoplightio/spectral/master/img/spectral-banner.png)
-
-This action uses [Spectral 4.x](https://github.com/stoplightio/spectral) to lint your OpenAPI files.
+This action uses [Spectral](https://github.com/stoplightio/spectral) from [Stoplight](https://stoplight.io/) to lint your OpenAPI documents, or any other JSON/YAML files.
 
 ![](./image.png)
+
+## Usage
+
+See [action.yml](action.yml)
+
+```yaml
+name: Run Spectral on Pull Requests
+
+on:
+  - pull_request
+
+jobs:
+  build:
+    name: Run Spectral
+    runs-on: ubuntu-latest
+    steps:
+      # Check out the repository
+      - uses: actions/checkout@v2
+
+      # Run Spectral
+      - uses: stoplightio/spectral-action@v0.5.5
+        with:
+          file_glob: 'doc/api/*.yaml'
+          spectral_ruleset: 'spectral:oas'
+```
+
+### Inputs
+
+- **file_glob:** Pattern describing the set of files to lint. Defaults to `*.oas.{json,yml,yaml}`. (_Note:_ Pattern syntax is documented in the [fast-glob](https://www.npmjs.com/package/fast-glob) package documentation)
+- **spectral_ruleset:** Ruleset to load in Spectral; either a built-in one or a custom one. Defaults to `spectral:oas`
+
+## Configuration
+
+Spectral Action will respect your [Spectral Rulesets](https://stoplight.io/p/docs/gh/stoplightio/spectral/docs/getting-started/rulesets.md), which can be defined, extended, and overriden by placing `.spectral.yml` in the root of your repository.
```

Comment:
@philsturgeon I'll reword a bit this section later

Reply:
@philsturgeon Or not... I've got kind of a writer block :-/

Reply:
This is great! Don’t worry about the text too much I’ll slap some polish on later.